blocks.js

```diff
@@ -2031,7 +2032,10 @@ SyntaxElementMorph.prototype.endLayout = function () {
     %shd    - Chameleon colored rectuangular drop-down for shadowed var names
     %lst    - chameleon colored rectangular drop-down for list names
     %b      - chameleon colored hexagonal slot (for predicates)
+<<<<<<< HEAD
```

Comment:
there are a bunch of these things in the code. Since I don't know anything about Git I'm scared....